microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/SimplifiedHealthTest.java

```diff
@@ -0,0 +1,84 @@
+package org.jboss.eap.qe.microprofile.health;
+
+import static io.restassured.RestAssured.get;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigurationException;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.restassured.http.ContentType;
+
+@RunAsClient
+@RunWith(Arquillian.class)
+public class SimplifiedHealthTest {
+
+    @Deployment(testable = false)
+    public static Archive<?> deployment() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, SimplifiedHealthTest.class + ".war")
```

Comment:
Optional - inline war. There is no need for creating war and it can be returned `ShrinkWrap.create(WebArchive.cl...`